index.html

```diff
@@ -111,6 +111,32 @@ <h2 class="ui divider horizontal">Services utilisant les API</h2>
   </p>
 </div>
 
+<section id="diff-data-api" class="ui container">
+  <div class="ui grid">
```

Comment:
> When possible, combine containers such as `container` and `grid` in a single `<div>` to avoid [divitis](http://www.apaddedcell.com/div-itis-what-it-and-how-avoid-it)  :)